ext/snowcrash

```diff
@@ -1 +1 @@
-Subproject commit 65e01df77c0e7ad86efb3c0efa4cbc69c846db98
+Subproject commit 76560473da89b5ba4be0a4826288ced2a9712533
```

Comment:
As I said, SC master is frozen and once this PR is merged, drafter master will also be frozen. This is what I mean by "preparing" releases. This way, we save some time when we are doing releases instead of doing more grunt work.